app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better if we implemented an `after_save` call back for this method. So that the method is automatically called whenever a like is saved. It would save us the stress of having to call it ourselves.

  ```
  after_save :update_likes_counter

  ...

  private
  
  def update_likes_counter
    post.increment!(:likes_counter)
  end
  ```